services/apiService.test.js

```diff
@@ -0,0 +1,56 @@
+import MOCK_DATA from '../mock-data/fellowship.json';
+import apiService from './apiService';
+
+const defaultFetchReference = global.fetch;
+describe('apiService', () => {
+
+  describe('getAllTeamMembers', () => {
+    beforeEach(() => {
+      global.fetch = jest.fn(() => Promise.resolve({
+        json: () => Promise.resolve(MOCK_DATA),
+      }));
+     });
+
+    afterEach(() => {
+      global.fetch = defaultFetchReference;
+    });
+
+    it('returns team memebers as an array of objects ', async () => {
+      const actual = await apiService.getAllTeamMembers('url');
+      expect(Array.isArray(actual)).toBe(true);
+    });
+
+    it('returns the object key for each team memeber as an id field in the team memebers data object key:{data} => {id:key, data}', async () => {
```

Comment:
This is a pretty long test description – I'd maybe simplify it to something like 'returns the team member's id as part of the data object'